### ISSUE TITLE
cmd/compile, importers: implement support for Alias types nodes in export data format

### ISSUE THREAD

**Issue by griesemer** [golang/go#64208]
This is a follow-up issue on #63223 to track the support for Alias types in importers.
@mdempsky 
@findleyr

**Comment by griesemer** [comment]
cc @mvdan

**Comment by mvdan** [comment]
Just checking in - is this still planned for Go 1.23? Once it lands in master, I'm more than happy to give it a try :)

**Comment by griesemer** [comment]
Yes. We're just super busy at the moment. Thanks.

**Comment by adonovan** [comment]
This has been done for ages, no?

**Comment by griesemer** [comment]
I believe so. Closing.
